rasa/importers/utils.py

```diff
@@ -0,0 +1,10 @@
+from typing import Iterable, Text
+
+from rasa.nlu.training_data import TrainingData
+
+
+def training_data_from_paths(paths: Iterable[Text], language: Text) -> TrainingData:
+    from rasa.nlu.training_data import loading
+
+    training_datas = [loading.load_data(nlu_file, language) for nlu_file in paths]
```

Comment:
`load_data` can throw a `ValueError` if the file does not exist. Can this happen here? Should we add a try/except`

Reply:
the file has to exist, cause we collected it earlier. There would have to be a delete right between finding and loading it. I don't think we'd have to check that extra.